CMakeLists.txt

```diff
@@ -0,0 +1,154 @@
+cmake_minimum_required(VERSION 3.27.6)
+
+project(elfshaker VERSION "0.0.17")
+
+enable_testing()
+
+include(FetchContent)
+set(FETCHCONTENT_QUIET OFF CACHE BOOL "" FORCE)
+
+#FetchContent_Declare(
+#    Corrosion
+#    GIT_REPOSITORY https://github.com/tipi-build/corrosion.git
+#    GIT_TAG c6132b645d1a0061a5c29ccf6a1d3c6f90490829 # v0.5 patched to support test execution, see https://github.com/tipi-build/corrosion/pull/1
+#)
+#FetchContent_MakeAvailable(Corrosion)
+set(CORROSION_VERBOSE_OUTPUT ON)
+add_subdirectory(cmake/corrosion)
```

Comment:
Why do you have remove fetchcontent and use submodule ?

Reply:
Let's explain you this in a private channel